canonical-webteam/unified-email-template
# Add the JAAS email receipt

## QA
- Download the [template from this PR](https://raw.githubusercontent.com/canonical-web-and-design/unified-email-template/8d2218d0f428e49f9ae8b2f0654f85bca3c47513/examples/jaas-reciept-email.html).
- Open the saved file in a browser
- See it looks like a receipt from JAAS
